packages/boot/test/bootstrapTests/vtransfer.test.ts

```diff
@@ -93,22 +97,32 @@ test('vtransfer', async t => {
   await EV(vtransferBridgeManager).fromBridge(expectedAckData);
 
   // verify the ackMethod outbound
-  const messages = getOutboundMessages(BridgeId.VTRANSFER);
+  const messages = outboundMessages.get(BridgeId.VTRANSFER);
   t.deepEqual(messages, [
-    {
-      target,
-      type: 'BRIDGE_TARGET_REGISTER',
-    },
+    { target, type: 'BRIDGE_TARGET_REGISTER' },
     {
       ack: btoa(JSON.stringify(expectedAckData)),
       method: 'receiveExecuted',
       packet,
       type: 'IBC_METHOD',
     },
   ]);
-
-  // test adding an interceptor for the same target, which should fail
-  await t.throwsAsync(() => evalProposal(testVtransferProposal), {
-    message: /Target.*already registered/,
-  });
+  await evaluateCoreProposal(testVtransferProposal);
+
+  /**
+   * test adding an interceptor for the same target, which should fail
+   * We could use `evaluateCoreProposal` here as well but the core eval
+   * failure is not critical so the failure is not propagated up
+   */
+  const coreEvalBridgeHandler = await EV.vat('bootstrap').consumeItem(
+    'coreEvalBridgeHandler',
+  );
+  await t.throwsAsync(
+    () =>
+      EV(coreEvalBridgeHandler).fromBridge({
+        evals: testVtransferProposal.evals,
+        type: QueuedActionType.CORE_EVAL,
+      }),
+    { message: /Target.*already registered/ },
+  );
```

Comment:
This second `evaluateCoreProposal(testVtransferProposal)` seems extraneous.
```suggestion
  // adding another interceptor for the same target should fail, but we must
  // errors in core evals are not propagated out so we must observe at a lower
  // level
  const coreEvalBridgeHandler = await EV.vat('bootstrap').consumeItem(
    'coreEvalBridgeHandler',
  );
  await t.throwsAsync(
    () =>
      EV(coreEvalBridgeHandler).fromBridge({
        type: QueuedActionType.CORE_EVAL,
        evals: testVtransferProposal.evals,
      }),
    { message: /Target.*already registered/ },
  );
```

Reply:
My bad